examples/md-flexible/MDFlexParser.cpp

```diff
@@ -478,7 +535,7 @@ double MDFlexParser::getDistributionMean() const { return distributionMean; }
 
 double MDFlexParser::getDistributionStdDev() const { return distributionStdDev; }
 
-std::string MDFlexParser::getWriteVTK() const { return writeVTK; }
+std::string MDFlexParser::getVTKFilenName() const { return vtkFileName; }
 
 double MDFlexParser::getBoxLength() {
```

Comment:
The behavior of this function should be documented, because it is unusual that a getter modifies attributes.